subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -434,7 +434,7 @@ int ull_adv_aux_init(void)
 	return 0;
 }
 
-int ull_adv_aux_reset(void)
+int ull_adv_aux_reset_finalize(void)
```

Comment:
I am not sure I understand this renaming: Without a `ull_adv_aux_reset`, or a `ull_adv_aux_reset_start` or anything like that, what purpose does the qualifier `finalize` have? 

I see that we have a `ull_adv_reset_finalize` but we also have a `ull_adv_reset` which doesn't seem to be the case for  `ull_adv_aux`

Reply:
`ull_adv_reset` is responsible for stopping all scheduling in ULL and request to stop scheduling LLL.
After that myflay is used to call LLL to reset all statics and globals.
Then `ull_adv_reset_finalize` enters to reset and initialize ULL globals.

I didn't change the naming conventions, just change other functions that actually do ULL memory clear and initialization.